i18n/en.json

```diff
@@ -1530,6 +1530,10 @@
     "translation": "Initializing open graph protocol api routes"
   },
   {
+    "id": "api.post.read_only_town_square",
```

Comment:
This should start with app instead of api

Reply:
@hmhealey I'll change this to be `api.post.create_post.town_square_read_only` since that's what was implemented on this PR: https://github.com/mattermost/mattermost-redux/pull/222/files
I used namespace `api` because all other error messages have the same values there.

Reply:
Ah, that's fine then